File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.js
function solution(n) {    //순서쌍의 개수
    let factors = [];
    
    for(let i = 1; i <= n / 2; i++){
        if(n % i === 0){
            factors.push(i);
        }
    }
    
    return factors.length + 1;
}

console.log(solution(100));

File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.go
func solution(n int) int { // 순서쌍의 개수
	var factors []int

	for i := 1; i <= n/2; i++ {
		if n%i == 0 {
			factors = append(factors, i)
		}
	}
	return len(factors) + 1
}